pkg/config/cert_rotation_config.go

```diff
@@ -29,15 +33,136 @@ type CertRotationConfig struct {
 
 	// Port is the port where the service runs.
 	Port string `env:"PORT,default=8080"`
+
+	// -- Crypto variables --
+	// KeyTTL is the length of time that we expect a key to be valid for.
+	KeyTTL time.Duration `yaml:"key_ttl,omitempty" env:"KEY_TTL,overwrite"`
+
+	// GracePeriod is a length of time between when we rotate the key and when an old Key Version is no longer valid and available
+	GracePeriod time.Duration `yaml:"grace_period,omitempty" env:"GRACE_PERIOD,overwrite"`
+
+	// PropagationDelay is the time that it takes for a change in the key in KMS to be reflected in the client.
+	PropagationDelay time.Duration `yaml:"propagation_delay,omitempty" env:"PROPAGATION_DELAY,overwrite"`
+
+	// DisabledPeriod is a time between when the key is disabled, and when we delete the key.
+	DisabledPeriod time.Duration `yaml:"disabled_period,omitempty" env:"DISABLED_PERIOD,overwrite"`
+
+	// TODO: This is intended to be temporary, and will eventually be retrieved from a persistent external datastore
+	// https://github.com/abcxyz/jvs/issues/17
+	// KeyName format: `projects/*/locations/*/keyRings/*/cryptoKeys/*`
+	// https://pkg.go.dev/google.golang.org/genproto/googleapis/cloud/kms/v1#CryptoKey
+	KeyNames []string `yaml:"key_names,omitempty" env:"KEY_NAMES,overwrite"`
 }
 
 // Validate checks if the config is valid.
-func (c *CertRotationConfig) Validate() error {
-	var merr *multierror.Error
+func (cfg *CertRotationConfig) Validate() error {
+	var err *multierror.Error
+
+	if cfg.KeyTTL <= 0 {
+		err = multierror.Append(err, fmt.Errorf("key ttl must be a positive duration, got %q", cfg.KeyTTL))
+	}
```

Comment:
Nit, prefer `got` shadow for easier copy-paste:

```suggestion
	if got := cfg.KeyTTL; got <= 0 {
		err = multierror.Append(err, fmt.Errorf("key ttl must be a positive duration, got %q", got))
	}
```